pkg/entry/mongo/document.go

```diff
@@ -121,13 +122,19 @@ func (d *JobLogDocument) Populate(ctx context.Context, ts time.Time, record map[
 	return d.generateObjectID()
 }
 
+func cleanLogContent(content string) string {
+	endOfLineRegexp := regexp.MustCompile(`(\n)+$`)
+	return endOfLineRegexp.ReplaceAllString(content, "")
```

Comment:
Tip, if you use regex, please compile them on start, not in each `cleanLogContent` calls.

Anyway, I suggest you to use: [`strings.TrimSuffix`](https://pkg.go.dev/strings#TrimSuffix)

```suggestion
	return strings.TrimSuffix(content, "\n")
```

PS Here only a single `\n` is stripped, IMO this is what we want.

Reply:
Fixed